packages/scheduler/npm/index.native.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+if (typeof global.nativeRuntimeScheduler !== 'undefined') {
```

Comment:
Why does this need `global` as opposed to just 

```js
if (typeof nativeRuntimeScheduler !== 'undefined') {
  module.exports = nativeRuntimeScheduler;
```

?

I know `global` should exist in RN environment but seems better if we don't have to rely on it?

Reply:
Let me remove it. I didn't realise it works even without `global` (I tested it). Thanks for the suggestion.